src/components/SpotlightCard/SpotlightCard.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import styles from './SpotlightCard.module.css'
+
+function SpotlightCard(props) {
```

Comment:
This is a nitpick, but I find writing props over and over again within one of these functional components to be a waste of use of my poor ol' tendons. I like to use destructuring of the props object when I know there will probably be more props attributes added by myself or others later.  I think it makes writing the types easier and it makes reading and writing the function easier too. 

Now that I'm going through more of the code, I'm wondering if there's a reason Next used the, uguuuu, dot-notation for all the styles in their original starter pack... I've mainly used css-in-js, and I'm not sure if that's a css-modules thing 🤔 

Anyway this is definitely not blocking merging and I wouldn't be surprised if Prettier can rewrite all of those into either pattern

Reply:
Good suggestion, I destructured the props. I haven't seen Prettier do that automatically, but I don't know if I'd want it to anyway, since sometimes I might find it overkill to destructure props.